### PR TITLE
As a user, after completing a game, I can choose to play again or quit

### PR DESCRIPTION
**Pull request by ellehallal**
- Displays play again prompt and exit message
- Board class: clear_squares method clears the marks from the board
- Controller: add play_again method. Returns true or false depending on user's input

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -34,4 +34,8 @@ def moves_remaining?
     available_squares = @squares.count { |square| square.is_a? Integer }
     available_squares > 0
   end
+
+  def clear_squares
```
Comment (matthewglover) [inline lib/board.rb:38]: This works, but why not just create a new empty board?

File: lib/controller.rb [hunk 2]
```diff
@@ -4,7 +4,22 @@ def initialize(game)
...
   def play
-    @game.play_move until @game.is_over?
-    @game.tie_or_won
+    play_game = true
+    while play_game
+      @game.play_move until @game.is_over?
+      @game.tie_or_won
+      play_game = play_again?
+    end
+    @game.display.show_exit_message
```
Comment (matthewglover) [inline lib/controller.rb:13]: This works, and shows you understand how to solve the problem. Which is the most important thing... but the play method now has two nested loop the `play_game` loop and the `play_move` loop. How do you test this? My suggestion is that these should be different objects not just different methods.

File: lib/controller.rb [hunk 3]
```diff
@@ -4,7 +4,22 @@ def initialize(game)
...
+  end
+
+  def play_again?
```
Comment (matthewglover) [inline lib/controller.rb:16]: The game is now definitely breaching SRP! All the bits you've done are right, but I think there should be more objects than just game :). What's clear is that @display doesn't fit within @game.

File: lib/board.rb (hunk 1 above)
Comment (ellehallal) [reply lib/board.rb:38]: Good point. Looks like I was overthinking this!

File: lib/controller.rb (hunk 3 above)
Comment (ellehallal) [reply lib/controller.rb:16]: I agree, display doesn't fit within Game. I'll change this. Thanks!

**Pull request merged**